clustered-indexes.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Clustered Indexes
+summary: Learn how clustered indexes apply to TiDB.
+---
+
+# Clustered Indexes
+
+> **Warning:**
+>
+> Clustered indexes is an experimental feature introduced in TiDB 5.0-rc.
+
+Clustered indexes provide the ability to organize tables in a way that can improve the performance of certain queries. The term _clustered_ in this context refers to the _organization of how data is stored_ and not _a group of database servers working together_. Some database management systems refer to clustered indexes as _index-organized tables_ (IOT).
+
+TiDB only supports clustering by a table's `PRIMARY KEY`. With clustered indexes enabled, the terms _the_ `PRIMARY KEY` and _the clustered index_ might be used interchangeably. `PRIMARY KEY` refers to the constraint (a logical property), and clustered index describes the physical implementation of how the data is stored.
+
+## Default behavior
+
+TiDB has always supported clustered indexes, provided the following criteria are true:
```

Comment:
I assume since before 1.0 (it has since I started using it). The differences are hilighted - in the situation where the _folllowing criteria are true_ TiDB 5.0 will not change anything for you.

Reply:
@coocood @bb7133 Could you help confirm when is `always`?

Reply:
That implies that it is a different type of clustered index, which is not the case. It has always clustered indexes provided certain criteria are true. Now it will cluster in all scenarios.

Reply:
```suggestion
Before v5.0, TiDB has only limited support for clustered indexes, provided the following criteria are true:
```